DocumentsFromSnapshotMigration/docker/entrypoint.sh

```diff
@@ -43,5 +43,6 @@ if [[ $RFS_COMMAND != *"--target-password"* ]]; then
     fi
 fi
 
-echo "Executing RFS Command"
-eval $RFS_COMMAND
+[ -z "$RFS_COMMAND" ] && \
+{ echo "Warning: RFS_COMMAND is empty! Exiting."; exit 1; } || \
+until ! { echo "Running command $RFS_COMMAND"; eval "$RFS_COMMAND"; }; do :; done
```

Comment:
We should definitely delete the S3/Lucene directories between runs.  This enables us stay without our disk space boundaries (while we care about them) as well having a clean and repeatable starting point for the process.  The only tradeoff I can think of is the need to re-download some small snapshot metadata files, which feels non-impactful.